src/rebar_state.erl

```diff
@@ -438,13 +438,13 @@ namespace(#state_t{namespace=Namespace}) ->
 namespace(State=#state_t{}, Namespace) ->
     State#state_t{namespace=Namespace}.
 
-packages(State=#state_t{packages=undefined}) ->
-    rebar_packages:get_packages(State);
-packages(#state_t{packages=Packages}) ->
-    Packages.
+packages_graph(State=#state_t{packages_graph=undefined}) ->
+    rebar_packages:get_packages_graph(State);
```

Comment:
Can't do that! This is the way to leak `ets` table.
Calling `ets:file2tab(SomeThing)`, more than once on the same file creates copy of that table.
See https://github.com/rebar/rebar3/pull/728.